src/tla/Apalache.tla

```diff
@@ -0,0 +1,63 @@
+--------------------------- MODULE Apalache -----------------------------------
+(*****************************************************************************)
+(* This is a standard module for use with the Apalache model checker.        *)
+(* The meaning of the operators is explained in the comments.                *)
+(* Many of the operators serve as additional annotations of their arguments. *)
+(* As we like to preserve compatibility with TLC and TLAPS, we define the    *)
+(* operator bodies by erasure. The actual interpretation of the operators is *)
+(* encoded inside Apalache. For the moment, these operators are mirrored in  *)
+(* the class at.forsyte.apalache.tla.lir.oper.BmcOper.                       *)
+(*                                                                           *)
+(* Igor Konnov, 2020                                                         *)
+(*****************************************************************************)
+
+(*****************************************************************************)
+(* An assignment of an expression e to a state variable x. Typically, one    *)
+(* uses the non-primed version of x in the initializing predicate Init and   *)
+(* the primed version of x (that is, x') in the transition predicate Next.   *)
+(* Although TLA+ does not have a concept of a variable assignment, we find   *)
+(* this concept extremely useful for symbolic model checking. In pure TLA+,  *)
+(* one would simply write x = e, or x \in {e}.                               *)
+(*                                                                           *)
+(* Apalache automatically converts some expressions of the form              *)
+(* x = e or x \in {e} into assignments. However, if you like to annotate     *)
+(* assignments by hand, you can use this operator.                           *)
+(*                                                                           *)
+(* For a further discussion on that matter, see:                             *)
+(* https://github.com/informalsystems/apalache/blob/ik/idiomatic-tla/docs/idiomatic/assignments.md *)
+(*****************************************************************************)
+x := e == x = e
+
+(*****************************************************************************)
+(* Annotating an expression \E x \in S: P as Skolemizable. That is, it can   *)
```

Comment:
This just sounds better to my hear. Not even sure why :) Feel free to disregard.

```suggestion
(* Annotates an expression \E x \in S: P as Skolemizable. That is, it can   *)
```